platform/src/test/java/org/hillview/test/dataset/BasicStatSketchTest.java

```diff
@@ -18,30 +18,34 @@
 package org.hillview.test.dataset;
 
 import org.hillview.dataset.api.IDataSet;
-import org.hillview.sketches.*;
+import org.hillview.sketches.BasicColStatSketch;
 import org.hillview.sketches.results.BasicColStats;
-import org.hillview.test.BaseTest;
-import org.hillview.utils.JsonList;
-import org.hillview.utils.TestTables;
+import org.hillview.sketches.results.HLogLog;
 import org.hillview.table.SmallTable;
 import org.hillview.table.Table;
 import org.hillview.table.api.ITable;
+import org.hillview.test.BaseTest;
+import org.hillview.utils.JsonList;
+import org.hillview.utils.Pair;
+import org.hillview.utils.TestTables;
 import org.junit.Assert;
 import org.junit.Test;
 
+import static org.junit.Assert.assertTrue;
```

Comment:
I don't really like static imports. Can we avoid them?

Reply:
I removed this static import. But searching "import static" returned 32 results in the whole project, mostly in test files. Maybe I'll make a separate PR to remove them all.